pkgs/tools/misc/grub/pvgrub_image/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, grub2_xen }:
+
+with stdenv.lib;
+let
+  efiSystemsBuild = {
+    "i686-linux".target = "i386";
+    "x86_64-linux".target = "x86_64";
+    "aarch64-linux".target = "aarch64";
+  };
+
+in (
+
+stdenv.mkDerivation rec {
+  name = "pvgrub-image";
+
+  configs = ./configs;
+
+  buildInputs = [ grub2_xen ];
+
+  buildCommand = ''
+    cp "${configs}"/* .
+    tar -cf memdisk.tar grub.cfg
+    # We include all modules except all_video.mod as otherwise grub will fail printing "no symbol table"
```

Comment:
@michalpalka I like content without words like "I believe" or "probably" better than content that just guesses something. 

While I understand it's hard to know everything, striving to documenting the reasons for failures makes it possible to improve things later on. 

It has already been merged, which is good for continuity, but if you do have a superior understand of exactly why things happen the way they do with all_video.mod, please document it.